pkg/k8s/factory_functions.go

```diff
@@ -154,3 +176,333 @@ func copyObjToV1Namespace(obj interface{}) meta_v1.Object {
 	}
 	return ns.DeepCopy()
 }
+
+func getVerStructFrom(objMeta meta_v1.Object) (versioned.UUID, versioned.Object) {
```

Comment:
Is this intentionally duplicated with the same-named function in `pkg/k8s/utils/`?

Reply:
I thought it would create import cyclo. I've move it to `pkg/k8s/util`